scripts/Epl-run.bat

```diff
@@ -321,7 +321,15 @@ IF EXIST SLABSurfaceTemps.TXT DEL SLABSurfaceTemps.TXT
 "%program_path%EnergyPlus"
 if %pausing%==Y pause
 
-:  5. Copy Post Processing Program command file(s) to working directory
+:  5.0 If native csv output is present, then skip convertESOMTR and ReadVarsESO post-processing
+IF EXIST eplusout.csv GOTO skipReadVars

```

Comment:
So these files are cleaned up prior to running E+, then if E+ produces them on its own, it is detected and ReadVars is skipped.  Sounds good.